### PR TITLE
chore(Team): Update

### PR DESCRIPTION
**Pull request by BloodhoundNL**
Updated

### CODE REVIEW

File: team.json [hunk 1]
```diff
@@ -111,34 +92,28 @@
           "country": "CY"
         },
         {
-          "firstName": "Michael “Furyo” Cabilao",
+          "firstName": "Daniel “IllustrisJack” Handayan Mayer",
```
Comment (IllustrisJack) [inline team.json:95]: ?

Comment (kaloczikvn) [reply team.json:95]: You were moved from Project Management to Development as far as I can see, is this info wrong? :O

Comment (IllustrisJack) [reply team.json:95]: yea i´m not an active member of the development team per say

Comment (IllustrisJack) [reply team.json:95]: maybe external consultant at best haha :D

**Pull request merged**